ansible/files/runtimes.json

```
@@ -208,6 +208,22 @@
             }
         ]
     },
+    "golang": [
```

this needs to be `go` instead
```suggestion
    "go": [
```